external-crates/move/move-execution/v1/crates/move-bytecode-verifier/src/code_unit_verifier.rs

```diff
@@ -25,7 +24,7 @@ use std::collections::HashMap;
 
 pub struct CodeUnitVerifier<'a> {
     resolver: &'a CompiledModule,
-    function_view: FunctionView<'a>,
+    function: FunctionContext<'a>,
```

Comment:
Just calling out: same nits in this file as in the v0 version here and on line 94.